databricks/koalas/tests/test_numpy_compat.py

```diff
@@ -52,3 +60,44 @@ def test_np_unsupported(self):
         kdf = self.kdf
         with self.assertRaisesRegex(NotImplementedError, "Koalas.*not.*support.*sqrt.*"):
             np.sqrt(kdf.a, kdf.b)
+
+    def test_np_spark_compat(self):
+        # Use randomly generated dataFrame
+        pdf = pd.DataFrame(
+            np.random.randint(-100, 100, size=(np.random.randint(100), 2)), columns=['a', 'b'])
+        kdf = ks.from_pandas(pdf)
+
+        blacklist = [
+            # Koalas does not currently support
+            "conj",
+            "conjugate",
+            "isnat",
+            'matmul',
+            'frexp',
```

Comment:
i wonder why `'matmul'` and `'frexp'` are only using single quote ??

is there any special purpose??

Reply:
Thanks, I will fix them.